kubernetes-cluster-autoscaler.tf

```diff
@@ -0,0 +1,138 @@
+locals {
+  cluster_autoscaler_chart_name = "cluster-autoscaler"
```

Comment:
Are all of these locals necessary? Some are only used once.

If going with this pattern, I think we should be consistent across all the addon files.

Reply:
in my opinion it's good practice to leave string literals etc out of the code

Reply:
But they are still in the code 😢